_posts/2018-08-04-pfsense-as-hvm-on-pc-engines-apu2.md

```diff
@@ -0,0 +1,562 @@
+Setting up harvesting: [UMA],[FS_ATIME],SWI,INTERRUPT,NET_NG,NET_ETHER,NET_TUN,MOUSE,KEYBOARD,ATTACH,CACHED
+Feeding entropy: dd: /entropy: Read-only file system
+dd: /boot/entropy: Read-only file system
+.
+Starting Network: lo0 igb0 enc0.
+lo0: flags=8049<UP,LOOPBACK,RUNNING,MULTICAST> metric 0 mtu 16384
+        options=600003<RXCSUM,TXCSUM,RXCSUM_IPV6,TXCSUM_IPV6>
+        inet6 ::1 prefixlen 128
+        inet6 fe80::1%lo0 prefixlen 64 scopeid 0x2
+        inet 127.0.0.1 netmask 0xff000000
+        nd6 options=21<PERFORMNUD,AUTO_LINKLOCAL>
+        groups: lo
+igb0: flags=8c02<BROADCAST,OACTIVE,SIMPLEX,MULTICAST> metric 0 mtu 1500
+        options=6403bb<RXCSUM,TXCSUM,VLAN_MTU,VLAN_HWTAGGING,JUMBO_MTU,VLAN_HWCSUM,TSO4,TSO6,VLAN_HWTSO,RXCSUM_IPV6,TXCSUM_IPV6>
+        ether 00:0d:b9:43:3f:bd
+        hwaddr 00:0d:b9:43:3f:bd
+        nd6 options=29<PERFORMNUD,IFDISABLED,AUTO_LINKLOCAL>
+igb0    media: Ethernet: link state changed to UP
+ autoselect (1000baseT <full-duplex>)
+        status: active
+enc0: flags=0<> metric 0 mtu 1536
+        nd6 options=29<PERFORMNUD,IFDISABLED,AUTO_LINKLOCAL>
+        groups: enc
+Starting devd.
+Starting Network: igb0.
+igb0: flags=8c02<BROADCAST,OACTIVE,SIMPLEX,MULTICAST> metric 0 mtu 1500
+        options=6403bb<RXCSUM,TXCSUM,VLAN_MTU,VLAN_HWTAGGING,JUMBO_MTU,VLAN_HWCSUM,TSO4,TSO6,VLAN_HWTSO,RXCSUM_IPV6,TXCSUM_IPV6>
+        ether 00:0d:b9:43:3f:bd
+        hwaddr 00:0d:b9:43:3f:bd
+        nd6 options=29<PERFORMNUD,IFDISABLED,AUTO_LINKLOCAL>
+        media: Ethernet autoselect (1000baseT <full-duplex>)
+        status: active
+Starting Network: enc0.
+enc0: flags=0<> metric 0 mtu 1536
+        nd6 options=29<PERFORMNUD,IFDISABLED,AUTO_LINKLOCAL>
+        groups: enc
+add host 127.0.0.1: gateway lo0 fib 0: route already in table
+add host ::1: gateway lo0 fib 0: route already in table
+add net fe80::: gateway ::1
+add net ff02::: gateway ::1
+add net ::ffff:0.0.0.0: gateway ::1
+add net ::0.0.0.0: gateway ::1
+Generating host.conf.
+eval: cannot create /etc/host.conf: Read-only file system
+eval: cannot create /etc/host.conf: Read-only file system
+eval: cannot create /etc/host.conf: Read-only file system
+Creating and/or trimming log files.
+Starting syslogd.
+Clearing /tmp (X related).
+Starting local daemons:/dev/md3: 8.0MB (16384 sectors) block size 32768, fragment size 4096
+        using 4 cylinder groups of 2.03MB, 65 blks, 384 inodes.
+super-block backups (for fsck_ffs -b #) at:
+ 192, 4352, 8512, 12672
+
+Welcome to pfSense!
+
+Please choose the appropriate terminal type for your system.
+Common console types are:
+   ansi     Standard ANSI terminal
+   vt100    VT100 or compatible terminal
+   xterm    xterm terminal emulator (or compatible)
+   cons25w  cons25w terminal
+
+Console type [vt100]:
+```
+
+Unfortunately pfSense had problem getting DHCP offer and didn't configure IP
+address - I tried to figure out what is wrong but my BSD-fu is low. I will
```

Comment:
BSD-fu?

Reply:
@miczyg1 BSD type of kung-fu :) `kung fu can refer to any discipline or skill achieved through hard work and practice`